packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -359,6 +359,10 @@ export const getDocLinks = ({ kibanaBranch }: GetDocLinkOptions): DocLinks => {
         macos_system_ext: `${SECURITY_SOLUTION_DOCS}deploy-elastic-endpoint.html#system-extension-endpoint`,
         linux_deadlock: `${SECURITY_SOLUTION_DOCS}ts-management.html#linux-deadlock`,
       },
+      packageActionTroubleshooting: {
+        // TODO: Pending to be updated when docs are ready
+        es_connection: `${SECURITY_SOLUTION_DOCS}`,
```

Comment:
Need docs link from docs team here

Reply:
> Need docs link from docs team here

@dasansol92 If we need new docs for troubleshooting this error, feel free to create a new docs issue in [elastic/security-docs](https://github.com/elastic/security-docs/issues) and assign it to me to get started. My guess is that we'll need to create a new section on the [Troubleshooting | Endpoint management](https://www.elastic.co/guide/en/security/master/ts-management.html) page, but I can confirm that as I learn more about the error and how it's resolved.

Reply:
@joepeeples Note I've removed the url for now (just let an empty string). This will hide the link button in the UI until we get the final one.